python/runAll.sh

```diff
@@ -95,25 +95,25 @@ fi
 #-------------------------------------------------
 #Set the environment for the batch job execution
 #-------------------------------------------------
-if [[ $whereToLaunch != "lxplus" ]]; then
-  cd $CMSSW_BASE/src/
-  if [[ $whereToLaunch == "pisa" ]]; then
-    source /afs/pi.infn.it/grid_exp_sw/cms/scripts/setcms.sh
-    export TMPDIR=$CMSSW_BASE/src/tmp
-    if ! [ -e $TMPDIR ]; then mkdir $TMPDIR; fi
-  else
-    source /swshare/psit3/etc/profile.d/cms_ui_env.sh
-    export LD_PRELOAD="libglobus_gssapi_gsi_gcc64pthr.so.0":${LD_PRELOAD}
-    export LD_LIBRARY_PATH=/swshare/glite/globus/lib/:/swshare/glite/d-cache/dcap/lib64/:$LD_LIBRARY_PATH
-  export LD_PRELOAD="libglobus_gssapi_gsi_gcc64pthr.so.0:${LD_PRELOAD}"
-  fi
-
-  export SCRAM_ARCH="slc5_amd64_gcc462"
-  source $VO_CMS_SW_DIR/cmsset_default.sh
-  eval `scramv1 runtime -sh`
-
-  cd -   #back to the working dir
-fi
+#if [[ $whereToLaunch != "lxplus" ]]; then
```

Comment:
Ok, it should be fine now.
https://github.com/perrozzi/Xbb/pull/75/commits/8f7e019c67afa25dc909d459ff91a2cae03bfd39 <https://github.com/perrozzi/Xbb/pull/75/commits/8f7e019c67afa25dc909d459ff91a2cae03bfd39>

> On 20 Apr 2016, at 10:46, silviodonato <notifications@github.com> wrote:
> 
> In python/runAll.sh <https://github.com/perrozzi/Xbb/pull/73#discussion_r60370805>:
> 
> > -    export TMPDIR=$CMSSW_BASE/src/tmp
> > -    if ! [ -e $TMPDIR ]; then mkdir $TMPDIR; fi
> > -  else
> > -    source /swshare/psit3/etc/profile.d/cms_ui_env.sh
> > -    export LD_PRELOAD="libglobus_gssapi_gsi_gcc64pthr.so.0":${LD_PRELOAD}
> > -    export LD_LIBRARY_PATH=/swshare/glite/globus/lib/:/swshare/glite/d-cache/dcap/lib64/:$LD_LIBRARY_PATH
> > -  export LD_PRELOAD="libglobus_gssapi_gsi_gcc64pthr.so.0:${LD_PRELOAD}"
> > -  fi
> > -
> > -  export SCRAM_ARCH="slc5_amd64_gcc462"
> > -  source $VO_CMS_SW_DIR/cmsset_default.sh
> > -  eval `scramv1 runtime -sh`
> > -
> > -  cd -   #back to the working dir
> > -fi
> > +#if [[ $whereToLaunch != "lxplus" ]]; then
> I would just need:
> if [[ $whereToLaunch == "pisa" ]]; then
> export TMPDIR=$CMSSW_BASE/src/tmp
> fi
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/perrozzi/Xbb/pull/73/files/cd9636bf1bb7b94ab9c756cd72559db1c8a3b158#r60370805>